test/prawn_pdf_formatter_test.rb

```diff
@@ -41,10 +41,15 @@ def test_render_pdf_basic
     data.column_names = %w[a b]
 
     data.to_prawn_pdf
+  end
 
-    expected_output = IO.read(File.join(__dir__, 'expected_outputs/prawn_pdf_formatter/pdf_basic.pdf.test')).bytes
+  def xtest_render_pdf_basic
```

Comment:
I'd prefer a comment as to why it's disabled here.  Someone might assume this was a typo and try to fix it not knowing it's intentionally disabled.  Maybe use the `skip` method in minitest?  

https://www.paperlesspost.com/blog/teams/skipping-tests-minitest/